mapserver/crdppf.map.mako

```diff
@@ -1441,39 +1451,51 @@ LAYER
   CLASS
     NAME "Pollué, investigation nécessaire"
     EXPRESSION /9903/
-    COLOR 0 0 255
-    OUTLINECOLOR 0 0 0
+    STYLE
+        COLOR 0 0 255
```

Comment:
Is your indent 2 spaces or 4 spaces long?

Reply:
Same down under, in the `STYLE` you mostly used 4 spaces and 2 spaces elsewhere (but this is not of top importance...)

Reply:
it's 4 now in VS Code. But there are still some old copy-paste left